src/Compilers/CSharp/Portable/Symbols/Source/ExplicitInterfaceHelpers.cs

```diff
@@ -17,21 +17,78 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
+#nullable enable
+    internal class ExplicitInterfaceMemberInfo(Binder binder, BindingDiagnosticBag diagnostics, ExplicitInterfaceSpecifierSyntax explicitInterfaceSpecifier, string name, string? aliasQualifier)
```

Comment:
>(Binder binder, BindingDiagnosticBag diagnostics, ExplicitInterfaceSpecifierSyntax explicitInterfaceSpecifier, string name, string? aliasQualifier) [](http://example.com/codeflow?start=46&length=147)

I think that implicit capturing negatively affects readability and maintenance of the code. Therefore, I recommend to not use primary constructors.